net.crossfireatlas.crossfire.yaml

```diff
@@ -0,0 +1,26 @@
+app-id: net.crossfireatlas.crossfire
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions: 
+command: net.crossfireatlas.crossfire.start.sh
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands: 
+      - /usr/lib/sdk/openjdk11/install.sh
+  - name: Crossfire_client
+    buildsystem: simple
+    build-commands: 
+      - install -D download /app/bin/jxclient.jar
+      - install -D net.crossfireatlas.crossfire.start.sh /app/bin/net.crossfireatlas.crossfire.start.sh
+    sources:
+      - type: file
+        url: https://sourceforge.net/projects/crossfire/files/jxclient/jxclient.jar/download
+        sha256: e6af744ea986da0991233787be6c9bc5535a7f3fe8ea682fe3284f1152e67b77
+      - type: file
+        path: net.crossfireatlas.crossfire.start.sh
+finish-args: 
```

Comment:
`finish-args` are typically above `modules`

Reply:
also missing `--share=ipc` which is required for x11. (I thought this was an error in the linter @barthalion ?)

Reply:
OK, thanks. I am adding this.

Reply:
this is not completely addressed. see https://github.com/flathub/flathub/pull/3757/files#r1058270721

Reply:
I'll reiterate what was said:

> `finish-args` are typically above `modules`